tests/e2e/accounts/multisig/account_test.go

```diff
@@ -0,0 +1,307 @@
+package multisig
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	bankv1beta1 "cosmossdk.io/api/cosmos/bank/v1beta1"
+	basev1beta1 "cosmossdk.io/api/cosmos/base/v1beta1"
+	"cosmossdk.io/core/header"
+	"cosmossdk.io/math"
+	multisigaccount "cosmossdk.io/x/accounts/defaults/multisig"
+	v1 "cosmossdk.io/x/accounts/defaults/multisig/v1"
+	accountsv1 "cosmossdk.io/x/accounts/v1"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func TestE2ETestSuite(t *testing.T) {
+	suite.Run(t, NewE2ETestSuite())
+}
+
+// TestFullMultisig creates a multisig account with 1 member, sends a tx, votes and executes it
+// then adds 2 more members and changes the config to require 2/3 majority (also through a proposal).
+// Finally it creates a proposal that won't pass.
+func (s *E2ETestSuite) TestFullMultisig() {
```

Comment:
personal preference: this is a lot included into a single test. Would it make sense to split this into a different tests?